test/bidir-full-sync-tests.js

```diff
@@ -47,6 +48,107 @@ test2('bidirectional full sync test', getClusterSizes(5), 20000,
 	})
 );
 
+test2('bidirectional full sync throttling test', getClusterSizes(3), 20000,
+    prepareCluster(function(t, tc, n) {
+        return [
+            dsl.assertStats(t, tc, n+1, 0, 0),
+            dsl.waitForEmptyPing(t, tc),
+
+            assertReverseFullSyncThrottling(t, tc, 0, n*3, 5),
+
+            dsl.assertStats(t, tc, n+1, 0, 0),
+        ];
+    })
+);
+
+// assert if the number of joins is throttled.
+function assertReverseFullSyncThrottling(t, tc, idx, count, expectedJoins) {
+    var functions = [];
+
+    //pause handling joins to stall reverse full syncs
+    functions.push(pauseHandlingJoins);
+    functions.push(dsl.waitForEmptyPing(t, tc));
+
+    // trigger the reverse full syncs
+    functions.push(triggerReverseFullSyncs(tc, idx, count));
+
+    // verify the number of join requests
+    functions.push(verify);
```

Comment:
It's not possible because the we still want to "hold" the join-requests and reply to them later. Good point about the secret sauce; I changed the overwrite and restore a little bit to make sure we don't loose it.